### PR TITLE
Import/export wrappers: Remove new lines from log errors

### PR DESCRIPTION
**Pull request by zoran15**
Remove new lines from log errors as only the first line is filtered by gcloud due to subsequent lines not having an appropriate prefix. Alternate solution where each line would have a prefix would be more effort to implement, so this simpler solution was chosen. Not many cases where multi line errors are returned from Daisy.

### CODE REVIEW

File: cli_tools/common/utils/logging/service/logger.go [hunk 1]
```diff
@@ -237,6 +237,12 @@ func (l *Logger) runWithServerLogging(function func() (*daisy.Workflow, error),
 	return logExtension, err
 }
 
+func removeNewLinesFromMultilineError(s string) string {
```
Comment (EricEdens) [inline cli_tools/common/utils/logging/service/logger.go]: unit test?

File: cli_tools/common/utils/logging/service/logger.go [hunk 2]
```diff
@@ -237,6 +237,12 @@ func (l *Logger) runWithServerLogging(function func() (*daisy.Workflow, error),
...
+	// first line in a multi error line is of "Multiple errors" type and doesn't need a separator
```
Comment (EricEdens) [inline cli_tools/common/utils/logging/service/logger.go]: I'm not familiar with the "multiple errors" type. Is that a golang thing? Daisy?

File: cli_tools/common/utils/logging/service/logger.go [hunk 3]
```diff
@@ -237,6 +237,12 @@ func (l *Logger) runWithServerLogging(function func() (*daisy.Workflow, error),
...
+	firstNewLineRemoved := strings.Replace(s, "\n", " ", 1)
+	return strings.Replace(firstNewLineRemoved, "\n", "; ", -1)
```
Comment (EricEdens) [inline cli_tools/common/utils/logging/service/logger.go]: minor: Any reason to not use `ReplaceAll`? (I had to read the source code of strings.Replace to figure out what `-1` meant.)

File: cli_tools/common/utils/logging/service/logger.go (hunk 2 above)
Comment (zoran15) [reply cli_tools/common/utils/logging/service/logger.go]: https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/daisy/error.go#L154

**Pull request merged**